src/neo/Ledger/Blockchain.cs

```diff
@@ -359,6 +357,8 @@ private VerifyResult OnNewBlock(Block block)
                         Self.Tell(unverifiedBlock, ActorRefs.NoSender);
                     block_cache_unverified.Remove(Height + 1);
                 }
+                // We can store the new block in block_cache and tell the new height to other nodes after Persist().
+                system.LocalNode.Tell(PingPayload.Create(Singleton.Height));
```

Comment:
```suggestion
                system.LocalNode.Tell(Message.Create(MessageCommand.Ping, PingPayload.Create(Singleton.Height)));
```

Reply:
Changed, now it use the message instead of payload